### PR TITLE
bugfix/utest_transportBrokerTests

### PR DESCRIPTION
**Pull request by brettywhite**
- needed to be run in main thread
- in testsendpacket, start up messenger

### CODE REVIEW

File: sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java [hunk 1]
```diff
@@ -27,68 +28,85 @@ private void sleep(){
 	}
 	
 	public void testStart(){
-		TransportBroker broker = new TransportBroker(mContext, SdlUnitTestContants.TEST_APP_ID,rsvp.getService());
-		assertTrue(broker.start());
-		broker.stop();
+		new Handler(Looper.getMainLooper()).post(new Runnable() {
```
Comment (joeygrover) [inline sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java]: I don't believe these all have to be on the main thread. I think they just need a looper. You will have to call looper.prepare. The reason it works with putting them on the main thread is that the main thread's looper is already prepared.

File: sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java [hunk 2]
```diff
@@ -27,68 +28,85 @@ private void sleep(){
...
+			@Override
+			public void run() {
+				TransportBroker broker = new TransportBroker(mContext, SdlUnitTestContants.TEST_APP_ID,rsvp.getService());
+				Handler handler = new Handler();
+				assertTrue(broker.start());
+				BluetoothAdapter adapter = BluetoothAdapter.getDefaultAdapter();
+				assertNotNull(adapter);
+				assertTrue(adapter.isEnabled());
+				broker.routerServiceMessenger = new Messenger(handler);
+				assertNotNull(broker.routerServiceMessenger);
```
Comment (joeygrover) [inline sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java]: Now this isn't actually testing anything.

**Comment by brettywhite** [comment]
good call @joeygrover.

File: sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java [hunk 3]
```diff
@@ -27,68 +28,84 @@ private void sleep(){
...
+
 	}
 	
 	public void testSendPacket(){
+		if (Looper.myLooper() == null) {
+			Looper.prepare();
+		}
+
 		TransportBroker broker = new TransportBroker(mContext, SdlUnitTestContants.TEST_APP_ID,rsvp.getService());
+		Handler handler = new Handler();
```
Comment (askirk) [inline sdl_android/src/androidTest/java/com/smartdevicelink/transport/TransportBrokerTest.java]: Is this ever used?

**Comment by brettywhite** [comment]
@askirk you were right, it wasnt needed

**Pull request merged**